### PR TITLE
API-40310: Modify Alert Threshold for `VBADocuments::SlackStatusNotifier` Job and Add Consumer Counts

### PR DESCRIPTION
**Pull request by kristen-brown**
## Summary
This work is NOT behind a feature toggle.

In this PR, the alert rate threshold for expired submissions has been increased for 3% to 20% to reduce the number of false positives that occur for this alert, especially overnight and on the weekends.

In addition, this PR adds consumer counts that align with the expired percentages for each consumer.

One small change is that the consumer details are now sorted by consumer name rather than expired rate. This was done to reduce the complexity of the code given the new consumer count additions, but can be reverted if anyone thinks we should spend the time on it. I would have to rework the method a bit to implement the sorting.

My team (Lighthouse Banana Peels) owns the `vba_documents` module.

## Related issue(s)
[API-40310](https://jira.devops.va.gov/browse/API-40310)

Screenshot 2024-09-24 at 10 18 04 AM

## Testing done
The unit tests were modified to align with the changes made to the job. I also tested the job with my local data.

## Screenshots
None

## What areas of the site does it impact?
This PR impacts the `vba_documents` module only, and specifically the `VBADocuments::SlackStatusNotifier` job.

## Acceptance criteria
- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation) – N/A
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable) – N/A
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature – N/A

## Requested Feedback
No specific feedback requested.

### CODE REVIEW

**Comment by kristen-brown** [comment]
Lint check is failing due to `webrick` vulnerability. Will wait until that vulnerability is resolved to merge.

File: modules/vba_documents/app/sidekiq/vba_documents/slack_status_notifier.rb
Comment (mathisto) [inline modules/vba_documents/app/sidekiq/vba_documents/slack_status_notifier.rb:74]: adventure-time-gif

**Review by mathisto — APPROVED** [review]
Beautiful work. @cilestin will be happy to see the counts added after the percentage.

**Review by ryan-mcneil — APPROVED** [review]

**Pull request merged**
